app/utils/RequestCountry.scala

```diff
@@ -0,0 +1,11 @@
+package utils
+
+import com.gu.i18n.CountryGroup
+import play.api.mvc.Request
+
+object RequestCountry {
+  implicit class RequestWithFastlyCountry(r: Request[_]) {
+    def getFastlyCountry: Option[CountryGroup] = r.headers.get("X-GU-GeoIP-Country-Code").flatMap(CountryGroup.byFastlyCountryCode)
+  }
+  implicit class AuthenticatedRequestWithIdentity(r: /*Auth*/ Request[_])
```

Comment:
What is this for?

Reply:
Ah good spot, it's left over from the class I took it from in contributions frontend. I'll remove it